File: Assignment 27/Program1.java
import java.util.*;

class Pattern
{
    public void Display(int iRow,int iCol)
    {
        int i=0,j=0;

        for(i=1;i<=iRow;i++)
        {
            for(j=1;j<=iCol;j++)
            {
                System.out.print("*\t");
            }
            System.out.println();

        }
    }
}
class Program1
{
    public static void main(String a[])
    {
        int iValue1=0,iValue2=0;

        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter the number of Rows:");
        iValue1=sobj.nextInt();

        System.out.println("Enter the number of Colomns:");
        iValue2=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue1,iValue2);
    }

}

File: Assignment 30/Program1.c
#include<stdio.h>

void Display(int iRow,int iCol)
{
    int i=0,j=0;
   

    for(i=1;i<=iRow;i++)
    {
        for(j=1;j<=iCol;j++)
        {
            while(j<=i)
            {
                printf("*\t");
                j++;
            }
            if(j>iCol)
            {
                break;
            }
            printf("#\t");

        }
        printf("\n");
    }
}

int main()
{
    int iValue1=0,iValue2=0;

    printf("Enter the number of Rows:");
    scanf("%d",&iValue1);

    printf("Enter the number of Cloumn:");
    scanf("%d",&iValue2);

    Display(iValue1,iValue2);
}